### ISSUE TITLE
Android: Onfido.start() throw on second call onwards

### ISSUE THREAD

**Issue by darcien** [onfido/react-native-sdk#122]
## TLDR

Only on Android, if the user try to start Onfido flow again after exiting the first one, user could not start a new flow.

## Error

```
You attempted to set the key `alpha2CountryCode` with the value `"AU"` on an object that is meant to be immutable and has been frozen.
```

## Repro

With this config:
```ts
const config = {
  flowSteps: {
    welcome: true,
    captureFace: {
      type: OnfidoCaptureType.VIDEO,
    },
    captureDocument: {
      docType: OnfidoDocumentType.DRIVING_LICENCE,
      countryCode: OnfidoCountryCode.AUS,
    },
  },
};
```

And rough pseudo code:
```ts
const onPress = () => {    
    try {
      await Onfido.start({...config, sdkToken: '*****'});
    } catch (err: unknown) {
      console.error({err});
      return;
    }
};

// render
<Button onPress={onPress} label="Start Flow" />
```

Steps:
1. Press start button, Onfido flow started.
2. Press back button in Onfido flow, see error logged with code `USER_LEFT_ACTIVITY`.
3. Press start button again, see reported error logged.

## Expected behavior

`Onfido.start()` should not throw on 2nd call onwards, needing an app restart to be able to start the flow again.

## Suspect code

Error might be caused by this lines where it tries to assign a value to the `alpha2CountryCode` key as stated in the error message when the config has been frozen in other part of the code.
https://github.com/onfido/react-native-sdk/blob/d030e9e62e5f6ebc43004a1feb81ee6ed1aefab0/js/Onfido.ts#L53-L55

## Version Info

@onfido/react-native-sdk: "7.3.0"

<details><summary>react-native info output</summary>

```
System:
    OS: macOS 13.1
    CPU: (8) arm64 Apple M1
    Memory: 69.31 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.14.0 - /var/folders/8h/xr5bhffj3rs_r0gry3n8khww0000gn/T/yarn--1676433534437-0.15033982869297646/node
    Yarn: 1.22.18 - /var/folders/8h/xr5bhffj3rs_r0gry3n8khww0000gn/T/yarn--1676433534437-0.15033982869297646/yarn
    npm: 8.3.1 - ~/Library/Caches/fnm_multishells/21026_1675393698425/bin/npm
    Watchman: 2023.02.13.00 - /opt/homebrew/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /Users/darcien/.rbenv/shims/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 22.2, iOS 16.2, macOS 13.1, tvOS 16.1, watchOS 9.1
    Android SDK: Not Found
  IDEs:
    Android Studio: 2021.2 AI-212.5712.43.2112.8815526
    Xcode: 14.2/14C18 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.12 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.1.0 => 18.1.0
    react-native: 0.70.5 => 0.70.5
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

</details>
